### PR TITLE
Update signal recovery calculation

### PR DESCRIPTION
**Pull request by 4rnaudB**
## Description
Previous signal loss maps were computed as follows:
phi = gradient * epi_TE, with the gradient in Hz

But phi should be
phi = 2pi * gradient / B0_map_thickness * epi_slice_thickness * epi_TE, with gradient in Hz

Note, in the code a  /pi was added because np.sinc() multiplies input by pi.

### CODE REVIEW

File: shimming-toolbox/shimmingtoolbox/shim/sequencer.py [hunk 1]
```diff
@@ -820,29 +821,29 @@ def calc_shimmed_anat_orient(self, coefs, list_shim_slice):
         nib.save(nii_shimmed_anat_orient, fname_shimmed_anat_orient)
 
     def _plot_static_signal_recovery_mask(self, unshimmed, shimmed_Gz, mask):
-    # Plot signal loss maps
-        def calculate_signal_loss(B0_map):
-            G = np.gradient(B0_map, axis = 2)
-            signal_map = abs(np.sinc(self.epi_te * G))
+        # Plot signal loss maps
+        def calculate_signal_loss(gradient):
+            slice_thickness = self.nii_anat.header['pixdim'][3]
```
Comment (po09i) [inline shimming-toolbox/shimmingtoolbox/shim/sequencer.py]: Does this include slice gap?

Comment (4rnaudB) [reply shimming-toolbox/shimmingtoolbox/shim/sequencer.py]: no, you're right I looked at an acquisition of 1mm with a slice gap of 1mm and pixdim[3] is 2

Comment (4rnaudB) [reply shimming-toolbox/shimmingtoolbox/shim/sequencer.py]: I don't see any parameter that gives the correct value though

Comment (po09i) [reply shimming-toolbox/shimmingtoolbox/shim/sequencer.py]: The JSON file should include a parameter that has slice gap information.

**Review by po09i — APPROVED** [review]

**Pull request merged**